client-side/src/component/Footer.jsx

```diff
@@ -0,0 +1,41 @@
+import { Box } from "@mui/material";
+import React from "react";
+import CustomButton from "./CustomButton.jsx";
+
+const Footer = ({ tasks }) => {
+  return (
+    <Box
+      style={{
+        marginTop: "20px",
+        marginBottom: "20px",
+        display: "flex",
+        justifyContent: "space-evenly",
+        flexWrap: "wrap",
+      }}
+    >
+      <CustomButton
+        type={"success"}
+        text={`Success Tasks: ${tasks.reduce(
+          (acc, task) => (task.completed ? acc + 1 : acc),
+          0
+        )}`}
+        dis={tasks.length === 0}
+      />
+      <CustomButton
+        type={"warning"}
+        text={`Pending Tasks: ${tasks.reduce(
+          (acc, task) => (!task.completed ? acc + 1 : acc),
+          0
+        )}`}
+        dis={tasks.length === 0}
+      />
+      <CustomButton
+        type={"primary"}
+        text={`ALl Tasks: ${tasks.length}`}
+        dis={tasks.length === 0}
```

Comment:
duplicated usage

can you initiate a constant with a good name like ` const hasTasks = !!tasks.length`

Reply:
first usage for number of task
second usage for disabled property